modules/menu/css/README.md

```diff
@@ -78,7 +92,38 @@ Menu item modifiers may be combined.
 </div>
 ```
 
-## Divider
+### Icons
```

Comment:
I had to add some extra markup and classes to replicate the way icons are displayed with the React Menu. I copied the markup structure rendered by the React Menu with separate spans used for the icon, text, and secondary icon (each span is sized using `flex`).

I've been trying to find a way to do this with less markup, but I haven't been successful so far. What do you think? Is this ok or should I keep digging?

Reply:
So unfortunately with the text overflow stuff, the label needs a container here unless we wanted to remove `display: flex` from the container `li` (which is even worse IMO).

Rather than using a wrapper for the icons, I would just add a class to them (i.e. `<SystemIcon icon={uploadCloudIcon} className="wdc-menu-item-icon" />`)

Reply:
Using CSS Icon instead of SystemIcon